File: Modul 2/Soal 05/PRAK205-NIM-Nama.py
import math

inputs = []

while True:
    line = input()
    if not line:
        break
    inputs.extend(line.split())
inputs = list(map(float, inputs))

A, B = inputs

C = math.sqrt((B * B) - (A * A))
kel = A + B + C
luas = (A * C / 2)

print(f"Alas = {C:.0f} cm")
print(f"Tinggi = {A:.0f} cm")
print(f"Keliling = {kel:.0f} cm")
print(f"Luas = {luas:.0f} cm^2")

File: Modul 2/Soal 05/PRAK205-NIM-Nama.c
#include <stdio.h>
#include <math.h>

int main() {
    float A, B;
    double C, keliling, luas;

    printf("");
    scanf("%f %f", &A, &B);

    C = sqrt((B * B) - (A * A));
    keliling = A + B + C;
    luas = (A * C / 2);

    printf("Alas = %.0f cm\n", C);
    printf("Tinggi = %.0f cm\n", A);
    printf("Keliling = %.0f cm\n", keliling);
    printf("Luas = %.0f cm^2\n", luas);

    return 0;
}